deploy-config/sandbox.yml

```diff
@@ -9,5 +9,10 @@ admin_base_url: https://notify-sandbox.app.cloud.gov
 redis_enabled: 1
 default_toll_free_number: "+18885989205"
```

Comment:
phone number is okay here?  Doesn't need to be a secret?

Reply:
In this case no, and [all of our production numbers are public as well](https://github.com/GSA/notifications-api/blob/main/docs/all.md#current-production-phone-numbers)